pulsar-broker/src/main/java/org/apache/pulsar/broker/TransactionMetadataStoreService.java

```diff
@@ -200,9 +208,9 @@ public CompletableFuture<Void> handleTcClientConnect(TransactionCoordinatorID tc
         return completableFuture;
     }
 
-    public CompletableFuture<TransactionMetadataStore> openTransactionMetadataStore(TransactionCoordinatorID tcId) {
-        final Timer brokerClientSharedTimer =
-                pulsarService.getBrokerClientSharedTimer();
+    public CompletableFuture<MutablePair<TransactionMetadataStore,
+            TransactionRecoverTracker>> openTransactionMetadataStore(TransactionCoordinatorID tcId) {
```

Comment:
Can we just add the `recoverTracker` as a param? So that we don't need to return a pair here.

Reply:
done.